src/server/pfs/s3/driver.go

```diff
@@ -54,47 +53,20 @@ func NewMasterDriver() *MasterDriver {
 	return &MasterDriver{}
 }
 
-func branchBucketName(b *pfs.Branch) string {
-	if b.Repo.Type == pfs.UserRepoType {
-		return fmt.Sprintf("%s.%s", b.Name, b.Repo.Name)
-	}
-	return fmt.Sprintf("%s.%s.%s", b.Name, b.Repo.Type, b.Repo.Name)
-}
-
-func projectBranchBucketName(b *pfs.Branch) string {
-	if b.Repo.Type == pfs.UserRepoType {
-		return fmt.Sprintf("%s.%s.%s", b.Name, b.Repo.Name, b.Repo.Project.Name)
-	}
-	return fmt.Sprintf("%s.%s.%s.%s", b.Name, b.Repo.Type, b.Repo.Name, b.Repo.Project.Name)
-}
-
 func (d *MasterDriver) listBuckets(pc *client.APIClient, r *http.Request, buckets *[]*s2.Bucket) error {
-	var isProjectAware = mux.Vars(r)[isProjectAwareVar] == isProjectAwareValue
-	repos, err := pc.ListRepoByType("") // get repos of all types
+	repos, err := pc.ListRepo() // get repos of all types
```

Comment:
`ListRepo` only returns user repos, which I think is what you intend — but the comment is now incorrect.